lib/widget/input/dropdown.dart

```diff
@@ -99,6 +98,13 @@ abstract class SelectOne extends StatefulWidget
     return null;
   }
 
+  void _setValue(dynamic value) {
+    _controller.maybeValue = value;
+    if (value is String && value == '') {
```

Comment:
See my comment on the Github issue. <Empty string> maybe an actual value users want to use, so we reset only on `null`, plus expose a clear() method for the user to clear out the value.